src/input/Binding.zig

```diff
@@ -280,8 +280,15 @@ pub const Action = union(enum) {
     scroll_page_fractional: f32,
     scroll_page_lines: i16,
 
-    /// Adjust an existing selection in a given direction. This action
-    /// does nothing if there is no active selection.
+    /// Adjust the current selection in a given direction. Does nothing if no 
+    /// selection exists.
+    ///
+    /// Arguments:
+    ///   - AdjustSelection: left, right, up, down, page_up, page_down, home, end,
```

Comment:
I don't think the type is helpful to end users. If you look at our config docs we tend to just give the valid values.